wukong/zookeeper.py

```diff
@@ -20,9 +20,7 @@ def _get_hosts_from_state(state):
         replicas = shard_data['replicas']
         for replica, replica_data in replicas.items():
             if replica_data['state'] == 'active':
-                node_url = replica_data['base_url'][:-5]  # strip /solr
-                node_url = node_url.replace('http://', '')
-                active_nodes.add(node_url)
+                active_nodes.add(replica_data['base_url'])
```

Comment:
Not sure what's going on here. Hoping it works on our old zook cluster :/

Reply:
There was another spot in code that was adding this `/solr` back in, so this should be a no-op. It also makes this more agnostic overall, as I believe that the path is configurable within solr